js/app/src/lite/Playground.svelte

```diff
@@ -60,65 +60,94 @@
 	});
 
 	function shortcut_run(e: KeyboardEvent): void {
-		if (e.key === "Enter" && (e.metaKey || e.ctrlKey)) {
-			e.preventDefault();
+		if (e.key == "Enter" && (e.metaKey || e.ctrlKey)) {
 			dispatch("code", { code });
+			e.preventDefault();
 		}
 	}
 
+	function handle_theme_mode(target: HTMLDivElement): "light" | "dark" {
+		const force_light = window.__gradio_mode__ === "website";
+
+		let new_theme_mode: ThemeMode;
+		if (force_light) {
+			new_theme_mode = "light";
+		} else {
+			const url = new URL(window.location.toString());
+			const url_color_mode: ThemeMode | null = url.searchParams.get(
+				"__theme"
+			) as ThemeMode | null;
+			new_theme_mode = theme_mode || url_color_mode || "system";
+		}
+
+		if (new_theme_mode === "dark" || new_theme_mode === "light") {
+			apply_theme(target, new_theme_mode);
+		} else {
+			new_theme_mode = sync_system_theme(target);
+		}
+		return new_theme_mode;
+	}
+
+	function sync_system_theme(target: HTMLDivElement): "light" | "dark" {
+		const theme = update_scheme();
+		window
+			?.matchMedia("(prefers-color-scheme: dark)")
+			?.addEventListener("change", update_scheme);
+
+		function update_scheme(): "light" | "dark" {
+			let _theme: "light" | "dark" = window?.matchMedia?.(
+				"(prefers-color-scheme: dark)"
+			).matches
+				? "dark"
+				: "light";
+
+			apply_theme(target, _theme);
+			return _theme;
+		}
+		return theme;
+	}
+
+	function apply_theme(target: HTMLDivElement, theme: "dark" | "light"): void {
+		const dark_class_element = is_embed ? target.parentElement! : document.body;
+		const bg_element = is_embed ? target : target.parentElement!;
+		if (theme === "dark") {
+			dark_class_element.classList.add("dark");
+		} else {
+			dark_class_element.classList.remove("dark");
+		}
+	}
+
+	let active_theme_mode: ThemeMode;
+	let parent_container: HTMLDivElement;
+
+	onMount(() => {
+		var code_editors = document.getElementsByClassName("code-editor");
+		for (var i = 0; i < code_editors.length; i++) {
+			code_editors[i].addEventListener(
+				"keydown",
+				shortcut_run as EventListener,
+				true
+			);
+		}
```

Comment:
This still has a problem that the shortcut key pressed on a single editor will trigger re-running all the mounted playgrounds on the page.
The event listener here should be attached only to the child `<Code />` in this component.

BTW, such "Enter+CMD" shortcut can be `<Code />` component's feature (maybe a new event dispatcher like `run_requested`?), WDYT @pngwn 

I think it can be a separate PR.